base/applications/network/net/cmdUser.c

```diff
@@ -188,7 +188,11 @@ GetTimeInSeconds(VOID)
     GetSystemTimeAsFileTime(&FileTime);
     Time.u.LowPart = FileTime.dwLowDateTime;
     Time.u.HighPart = FileTime.dwHighDateTime;
-    RtlTimeToSecondsSince1970(&Time, &dwSeconds);
+    if (!RtlTimeToSecondsSince1970(&Time, &dwSeconds))
```

Comment:
Regarding `RtlTimeToSecondsSince1970`, and the related `RtlTimeToSecondsSince1980` and `RtlTimeFieldsToTime`: in the latest Windows WDK (see its ntifs.h file), these functions are not marked with `_Must_inspect_result_` (or `_Check_result_`): they are just marked with `_Success_(return != 0)` . So I don't know why in ReactOS they've got that extra `_Must_inspect_result_` thing.

Reply:
Actually, in ReactOS, `RtlTimeToSecondsSince1980` doesn't have the `_Must_inspect_result_` annotation, exactly as in the MS WDK. So whoever added it for `RtlTimeToSecondsSince1970`, besides breaking compatibility with MS WDK, also added an inconsistency.

Reply:
Should we remove those annotations, though ? Maybe we can add some define ?
`__REACTOS_NO_EXTRA_SAL_ANNOTATIONS__` maybe. We will have trouble with fastfat_new otherwise, and I don't feel like adding extra code just for this.

Reply:
Not sure it's very important to have `_Must_inspect_result_` for `RtlTimeToSecondsSince1970` just for ReactOS...

Reply:
Yeah, let's ditch it.